src/MRA.Jobs.Api/WebUI/appsettings.json

```diff
@@ -1,4 +1,8 @@
 {
+  "JwtSettings": {
```

Comment:
It should not be here. In production we will get it from azure key vault

Reply:
You need to get it from appsettings.Development.json

Reply:
Ok